notebooks/filtering.ipynb

```diff
@@ -5,40 +5,567 @@
    "id": "78cee78b",
```

Comment:
<p>Suggest using <a href="https://mystmd.org/guide/admonitions" target="_blank">Callouts &amp; Admonitions </a>to flag that this is an interactive activity for the learner.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ICESAT-2HackWeek/icesat2-cookbook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>